package.json

```diff
@@ -450,6 +450,16 @@
           "default": false,
           "description": "Run 'go test -coverprofile' on save"
         },
+        "go.showCovered": {
```

Comment:
@ramya-rao-a Just as well! I won't be able to get around to it until Friday though